gitpod-io/gitpod
# Update ssh gw dashboard

## Description


Simplify it to show most important info:
- throughput of valid and suspicious traffic
- total failure ratios and error ratios by kind

## Related Issue(s)

Fixes #

## How to test


See https://grafana.gitpod.io/d/anton-ssh/antons-ssh-gateway-overview?orgId=1&refresh=30s

## Release Notes

```release-note
```

## Documentation
